src/main.js

```diff
@@ -186,17 +186,26 @@ function takeMeBack() {
   savedPostersView.classList.add('hidden');
 }
 
+function pushValues() {
```

Comment:
Nice - let's test this out next.

Reply:
✅ I think that would wrap up iteration 2!